### DISCUSSION TITLE
No access to GitHub Copilot found

### DISCUSSION THREAD

**Discussion by xiaoswan** [community/community#112196] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/103245169/43c3db1b-b2b8-4baa-a621-d7f628bace49)
I have received emails that I have access to github copilot but cannot access

**Comment by xiaoswan** [comment]
![image](https://github.com/community/community/assets/103245169/84ea0a6f-b06c-4bb1-ab2d-9ab8b727a468)
This is my current state

**Reply by davecheney** [reply, marked as answer]
@xiaoswan there is no copilot subscription associated with this GitHub username.

**Reply by davecheney** [reply]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in with a different username

**Comment by gerteck** [comment]
Have you linked Visual Studio Code to your GitHub account? Hope it helps!

> If you have not previously authorized Visual Studio Code in your GitHub account, you will be prompted to sign in to GitHub in Visual Studio Code.

Referring to this [page](https://docs.github.com/en/copilot/using-github-copilot/getting-started-with-github-copilot).

**Reply by xiaoswan** [reply]
yes.How do I log into the organization account to use this?Or I need to bill for Github copilot
![image](https://github.com/community/community/assets/103245169/d21da092-a464-4a15-9d95-336b13eb893f)

**Reply by davecheney** [reply]
You should check with the org admin who invited you to their organisation. If you maintain multiple github accounts, they may have unintentionally invited the wrong one. Either ask them to invite @xiaoswan or log out and log in as the account which has been invited to the organisation.
